src/misc.jl

```diff
@@ -5,7 +5,9 @@ handle("ping") do
 end
 
 handle("args") do args
-  append!(ARGS, args)
+  if length((!isempty).(args)) == 0
+    append!(Base.ARGS, args)
+  end
```

Comment:
sorry for commenting after merge, but does this need to be 
```julia
if length(filter!(!isempty, args)) !== 0
  append!(Base.ARGS, args)
end
```

?

Reply:
Yeah, or `sum` instead of `length`. Will push a commit.